build/ci-build.yml

```diff
@@ -85,26 +95,24 @@ stages:
               command: 'custom'
               custom: 'new'
               arguments: '-i "templates/*.nupkg"'
-          - task: DotNetCoreCLI@2
-            displayName: 'Create new project from template'
-            inputs:
-              command: 'custom'
-              custom: 'new'
-              arguments: 'arcus-webapi -n MyProject -o /MyProject'
-          - task: Docker@1
-            displayName: 'Build Docker image from newly created web API project from template'
-            inputs:
-              dockerFile: /MyProject/Dockerfile
-              imageName: 'myproject:$(Build.BuildId)'
-          - task: Docker@1
-            displayName: 'Run new web API project Docker image from template'
-            inputs:
-              command: 'Run an image'
-              imageName: 'myproject:$(Build.BuildId)'
-              containerName: 'myproject'
+          - template: 'templates/run-new-project-from-template.yml'
+            parameters:
+              projectName: 'Arcus.Demo.WebApi'
+              projectAlias: 'arcus-webapi'
               ports: '$(Http.Port):$(Http.Port)'
               envVars: |
                 ARCUS_HTTP_PORT=$(Http.Port)
+          - template: 'templates/run-new-project-from-template.yml'
+            parameters:
+              projectName: 'Arcus.Demo.ServiceBus.Queue'
+              projectAlias: 'arcus-servicebus-queue'
+              ports: '$(Arcus_Worker_HealthPort):$(Arcus_Worker_HealthPort)'
+              envVars: |
+                ARCUS_HEALTH_PORT=$(Arcus_Worker_HealthPort)
+                EVENTGRID_TOPIC_URI=$(Arcus_Worker_EventGrid_TopicUri)
```

Comment:
Have now used some of my own Azure resources because I haven't access to the Arcus Azure subscription.

Reply:
You do have access ;) it's under the Codit tenant.

Reply:
Yeah, access; but can't create resource groups or have any idea where the resources should be located or configured. + indeed, the group variables and whereter or not we can re-use some resources from other groups.

Reply:
Ok, I think I fixed it. Now, only the Arcus Azure resources are used.

Reply:
> Yeah, access; but can't create resource groups or have any idea where the resources should be located or configured. + indeed, the group variables and whereter or not we can re-use some resources from other groups.

No need for more RGs (can't create them as well), but you have access to the current ones called `arcus-*`